components/eamxx/docs/user/coarse_nudging.md

```diff
@@ -0,0 +1,25 @@
+# Nudging from coarse data
+
+Because EAMxx is designed to run at ultra high resolution, it is not feasible to produce nudging data at the same resolution.
+Instead, in EAMxx, it is possible to nudge from coarse data.
+This is done by remapping the coarse data provided by the user to the runtime physics grid of EAMxx.
+In order to enable nudging from coarse data, the user must provide nudging data at the coarse resolution desired and an appropriate     ncremap-compatible mapping file.
+
+## Example setup
+
+A user can produce coarse nudging data from running EAMxx or EAM at a ne30pg2 or any other applicable resolution.
+Additionally, several users in the E3SM projects have produced nudging data at the ne30pg2 resolution from the MERRA2 and ERA5 datasets.
+A limitation for now is that the nudging data must be provided explicitly, either as one file or as a list of files.
+This can be problematic for long list of files, but we are working on a solution to this problem.
```

Comment:
ah, yes, that's true, we probably are constrained in the length of a string, either in python, bash, or XML, not sure which one.

Reply:
I _really_ need to learn to read the _whole_ message before replying. This sparse-read feature of my brain is not working well.

Since you noticed the error at run time, it may not be an issue of bash/python/xml, but rather something else. There shouldn't be any restriction to a std::string size that could hurt us here though (there is a hard limit, but it's like in the billions of chars...

Reply:
does EKAT have a hard limit?  And can't we switch to a vector of strings rather than one long string to parse?

Reply:
I do not, but I believe our restart files are rather large.  So we should be able to handle O(100GB), @bartgol (?).

That being said, I wonder if there is a performance cost in reading in such single large files, rather than a number of smaller ones. I'm inclined to believe that the overhead of opening a file using SCORPIO outweighs opening a larger file.

Reply:
Bigger picture, this still concerns me.  We should be this limited in the number of files we can feed to nudging.  I'm going to try to create a simple recreator and try to figure out the issue.